java/src/jmri/jmrix/mrc/MrcMessage.java

```diff
@@ -339,8 +339,9 @@ public int getAccState() {
 
     /**
      * set the fast clock ratio ratio is integer and max of 60 and min of 1
+     * @param ratio value to set new clock speed
      *
-     * @return MrcMessage
+     * @return MrcMessage new message to set the clock speed ratio
```

Comment:
You do not want to include the returned object class as the first item in the @return tag. It is automatically included by the Javadoc processor.